game/src/sandbox/time_warp.rs

```diff
@@ -92,14 +84,6 @@ impl JumpToTime {
                 ),
                 Line("minute delay").small_heading().into_widget(ctx),
             ]),
-            Toggle::checkbox(
-                ctx,
-                "skip drawing (for faster simulations)",
-                None,
-                app.opts.dont_draw_time_warp,
-            )
-            .margin_above(30)
-            .named("don't draw"),
             build_jump_to_delay_button(ctx, app.opts.jump_to_delay),
         ]);
 
```

Comment:
I can't comment on it, but the `exact_size_percent` below now seems a little excessive vertically on my smaller screen:
![Screenshot from 2021-05-11 17-40-02](https://user-images.githubusercontent.com/1664407/117901371-0edf3800-b280-11eb-8e84-9efe7dec9819.png)
Removing it entirely is weird, because the panel changes size between the two tabs. I'm fine leaving this as-is; I think you previously cleaned up lots of sizing logic based on percentage of the screen dims. There may be more polish we could apply everywhere, later.